papermage_components/utils.py

```diff
@@ -151,8 +151,8 @@ def get_table_image(table_entity: Entity, doc: Document, page_image=None):
     page_w, page_h = page_image.size
     table_image = page_image.crop(
         (
-            box.l * page_w,
-            box.t * page_h,
+            (box.l-0.01) * page_w,
```

Comment:
Could we not hardcode this value in? If it's something that's necessary to the new function of the table transformer, we should pass it in as a parameter. In any case, the reason for the choice of 0.01 as the value should be documented.

Reply:
Resolved. 0.01 should generally work for all pdfs, as papermage returns box.l and box.t as the relative lengths to the entire page